core/src/main/scala/scala/collection/parallel/Tasks.scala

```diff
@@ -180,15 +180,16 @@ trait AdaptiveWorkStealingTasks extends Tasks {
     def spawnSubtasks() = {
       var last: AWSTWrappedTask[R, Tp] = null
       var head: AWSTWrappedTask[R, Tp] = this
-      do {
+      while ({
         val subtasks = head.split
         head = subtasks.head
         for (t <- subtasks.tail.reverse) {
           t.next = last
           last = t
           t.start()
         }
-      } while (head.body.shouldSplitFurther)
+        head.body.shouldSplitFurther
+      }) ()
```

Comment:
Because I am a person,
```
-      do {
+      var more = true
+      while (more) {
-      } while (head.body.shouldSplitFurther)
+        more = head.body.shouldSplitFurther
+      }
```
I think block conditional will be more natural in scala 3 because no brackets are involved.

But covid is also perfectly natural.

Reply:
six of one, half a dozen of the other 🤷